chart/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for ai-rhdh-installer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+debug:
+  ci: false # Sets up the products for CI testing.
+  container: false # Creates a long running container to easily open a terminal for troubleshooting.
+  script: false # Prints commands before executing them.
+openshift-gitops:
+  enabled: true
+  channel: gitops-1.12
+  workload-namespaces:
+    - default
+  git-token: |- # Token to create/push repositories.
+    ${GITOPS__GIT_TOKEN}
+  argoCD:
+    # ArgoCD's application-controller resource limits.
+    controller:
+      resources:
+        limits:
+          cpu: "2"
+          memory: 6Gi
+        requests:
+          cpu: "1"
+          memory: 3Gi
+openshift-pipelines:
+  enabled: true
+  channel: pipelines-1.14
+
+# Private values
+# You may want to override those variables in another file (e.g. values-private.yaml)
+developer-hub:
```

Comment:
This file I believe is for configuration, which is our #3?

Reply:
You're right `values.yaml` will be used as part of #3 for configuration. I added it now so it is present as the subscriptions for the operators reference some items from it (default values set). I was thinking for #3 it would utilize this yaml file and can helm upgrade pieces of it for more configuration.